src/audiounit.cs

```diff
@@ -39,6 +39,8 @@ namespace XamCore.AudioUnit {
 			uint frameCount, nint inputBusNumber, AudioBuffers inputData);
 
 	delegate void AUScheduleParameterBlock (AUEventSampleTime eventSampleTime, uint rampDurationSampleFrames, ulong parameterAddress, float value);
+	[iOS (11, 0), Mac (10,13), TV (11,0), NoWatch]
+	delegate int AUMidiOutputEventBlock (long eventSampleTime, byte cable, nint length, ref byte midiBytes);
```

Comment:
incorrect,
`typedef OSStatus (^AUMIDIOutputEventBlock)(AUEventSampleTime eventSampleTime, uint8_t cable, NSInteger length, const uint8_t *midiBytes);`

The last argument is *not* a single byte, but a pointer to an array of bytes.